services/net/filemonitor/FilemonitorAction.cs

```diff
@@ -678,6 +680,16 @@ private static string GetItemSourceCode(IngestModel ingest, string paperName, Di
             var xmlTxt = ReadFileContents(filePath, ingest);
             if (xmlTxt != null)
             {
+                // Files regularly have the 'last' story corrupting in the <story> node.
+                // This results in the whole file being invalid.
+                // Check for and fix the common failure.
+                if (!_storyTag.Match(xmlTxt).Success)
+                {
+                    xmlTxt = $"{xmlTxt}</story><date>{DateTime.Now:MM-dd-yyyy}</date></bcng>";
```

Comment:
Append commonly missing data.
This does however mean we'll be missing content in this story and possibly any other stories that should be below this story.